diff --git a/node_modules/react-native-maps/lib/MapMarker.js b/node_modules/react-native-maps/lib/MapMarker.js
index 90b689e..d22177f 100644
--- a/node_modules/react-native-maps/lib/MapMarker.js
+++ b/node_modules/react-native-maps/lib/MapMarker.js
@@ -29,7 +29,7 @@ const react_native_1 = require("react-native");
 const decorateMapComponent_1 = __importStar(require("./decorateMapComponent"));
 const MapMarkerNativeComponent_1 = require("./MapMarkerNativeComponent");
 class MapMarker extends React.Component {
-    getNativeComponent;
+    // getNativeComponent;
     getMapManagerCommand;
     getUIManagerCommand;
     static Animated;
